### DISCUSSION TITLE
Curious

### DISCUSSION THREAD

**Discussion by onojaaofficial** [Cyfrin/foundry-full-course-cu#2883] [Q&A]
Hi everyone, I recently stumbled upon the "blockchain, smart contracts, Solidity" lesson. Since I am interested in learning Web3 due to the opportunities it will afford me, my question to those who are advanced in the field is: Is it normal to be confused during the lesson? I hear all that has been said, but I don't understand a lot of what is being said. Is this normal, or am I just dumb?

**Comment by EngrPips** [comment]
It is very normal, and I still get confused from time to time. You just need to take a step back and rethink everything, and maybe take a short break if things become very confusing. The moment it starts getting complicated, stop right there and take all the time you need to make sense of everything before you move on.

**Comment by adeyemijosh** [comment]
It is normal to get confused, but since you are new i will suggest getting familiar with how the technology works that is, getting to know the terms, the front end and how it works so when you start learning it will be easy for you to get around and yes, Just as EngrPips said take some break and get at it again

**Comment by chainflash** [comment]
Yes, it is completely normal, reviewing and repeating the content is very important.

**Comment by AberTheCreator** [comment]
It is normal to get confused when just getting Started,  you would get the context of everything as you go on.
